src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -14,6 +14,43 @@ public static class ThreadPoolTests
         private const int ExpectedTimeoutMilliseconds = ThreadTestHelpers.ExpectedTimeoutMilliseconds;
         private const int MaxPossibleThreadCount = 0x7fff;
 
+        static ThreadPoolTests()
+        {
+            // Run the following tests before any others
+            ConcurrentInitializeTest();
+        }
+
+        [Fact]
```

Comment:
Does this need to be marked [Fact]?  As it, it seems like it'll be invoked twice, once from the static cctor, and once as its own dedicated test.